test.py

```diff
@@ -115,6 +116,16 @@ def test_validate_fast(self):
                     "2478433644_2839c5e8b8_o_d.jpg"))
         self.assertRaises(bagit.BagValidationError, self.validate, bag, fast=True)
 
+    def test_validate_completeness(self):
```

Comment:
Since we have mock available, do you think it would be worth adding something like a mock.patch call for the full file content check so we can assert that it's not called?